fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1065,7 +1069,9 @@ private void handleAnalyzeProfileStmt() throws IOException {
         List<Integer> planNodeIds = analyzeProfileStmt.getPlanNodeIds();
         ProfileManager.ProfileElement profileElement = ProfileManager.getInstance().getProfileElement(queryId);
         Preconditions.checkNotNull(profileElement, "query not exists");
-        handleExplainStmt(ExplainAnalyzer.analyze(profileElement.plan, profileElement.profile, planNodeIds));
+        handleExplainStmt(ExplainAnalyzer.analyze(profileElement.plan,
+                RuntimeProfileParser.parseFrom(CompressionUtils.gzipDecompressString(profileElement.profileContent)),
```

Comment:
`ProfileEelement.profile` is removed in this pr, because it will take a lot of memory, average 2M for a query, the fe will take additional 3G memory  after executing 500 queries.

And we get the RuntimeProfile instance by parsing the profile content. The total time usage is around 1ms, which is totally acceptable in practical situations. because there won't be so many `analyze profile` statement at one time.

Reply:
can we keep small profile (e.g. bytes_size < 64KB) in memory for at most ttl time?  only the large profile is compressed and dump onto disk?